frontend/public/components/namespace.jsx

```diff
@@ -95,12 +96,20 @@ const ProjectRow = ({obj: project}) => {
   </ResourceRow>;
 };
 
-const ProjectEmptyMessageDetail = <p>
-  Create a project for your application. To learn more, visit the OpenShift <a href={openshiftHelpBase} target="_blank" rel="noopener noreferrer">documentation</a>.
-</p>;
-const ProjectEmptyMessage = () => <MsgBox title="Welcome to OpenShift" detail={ProjectEmptyMessageDetail} />;
-export const ProjectList = props => <List {...props} Header={ProjectHeader} Row={ProjectRow} EmptyMsg={ProjectEmptyMessage} />;
-export const ProjectsPage = props => <ListPage {...props} ListComponent={ProjectList} canCreate={true} createHandler={createProjectModal} />;
+const ProjectListComponent = props => {
+  const ProjectEmptyMessageDetail = <p>
+    {props.createProjectMessage || 'Create a project for your application.'} To learn more, visit the OpenShift <a href={openshiftHelpBase} target="_blank" rel="noopener noreferrer">documentation</a>.
```

Comment:
Suggest `white-space: pre-line;` or `white-space: pre-wrap;` on the message since it can have multiple lines.

I'd probably move the doc link into a separate paragraph when there's a custom message as well so they don't run together.

Ideally we'd linkify the message, but I don't think we have a library for that right now.